components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -262,6 +262,13 @@ func Run(options ...RunOption) {
 			WorkspaceID:       cfg.WorkspaceID,
 			SupervisorVersion: Version,
 		}, tokenService)
+
+		ownerID, err := gitpodService.GetOwnerID(ctx)
+		if err != nil {
+			log.WithError(err).Error("gitpod perf analytics: failed to resolve workspace info")
+		}
+
+		cfg.OwnerId = ownerID
```

Comment:
I don’t config should be changed like that, it is loaded and validated in config.go